commands/config_test.go

```diff
@@ -143,6 +143,49 @@ func (s *MmctlUnitTestSuite) TestConfigGetCmd() {
 		s.Require().Len(printer.GetLines(), 0)
 		s.Require().Len(printer.GetErrorLines(), 0)
 	})
+
+	s.Run("Get value if the key points to a map element", func() {
```

Comment:
Please add more tests here:
- For the `PlugginSettings.Plugins` case with: single value, slice or even another map